src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2122 @@
+                    Log.Error(LogTag, "Remove PreResult Failed with error " + error);
+                }
+
+                _preResult -= value;
+            }
+        }
+
+        /// <summary>
+        /// Called when client gets the specific engine's result from vc-service.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <pre>
+        /// The State should be Initialized
+        /// </pre>
+        public static event EventHandler<SpecificEngineResultEventArgs> SpecificEngineResult
+        {
+            add
+            {
+                _specificEngineResultDelegate = (IntPtr engineAppId, IntPtr evt, IntPtr result, IntPtr userData) =>
+                {
+                    string engineAppIdString = Marshal.PtrToStringAnsi(engineAppId);
+                    string eventString = Marshal.PtrToStringAnsi(evt);
+                    string resultString = Marshal.PtrToStringAnsi(result);
+                    SpecificEngineResultEventArgs args = new SpecificEngineResultEventArgs(engineAppIdString, eventString, resultString);
+                    _specificEngineResult?.Invoke(null, args);
+                };
+                ErrorCode error = VcMgrSetSpecificEngineResultCb(_specificEngineResultDelegate, IntPtr.Zero);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Add SpecificEngineResult Failed with error " + error);
+                }
+                else
+                {
+                    _specificEngineResult += value;
+                }
+            }
+
+            remove
+            {
+                ErrorCode error = VcMgrUnsetSpecificEngineResultCb();
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Remove SpecificEngineResult Failed with error " + error);
+                }
+
+                _specificEngineResult -= value;
+                _specificEngineResultDelegate = null;
+            }
+        }
+
+        /// <summary>
+        /// Event to be invoked when the recognition is done.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <pre>
+        /// The State should be Initialized
+        /// </pre>
+        public static event EventHandler<RecognitionResult> RecognitionResult
```

Comment:
```suggestion
        public static event EventHandler<RecognitionResult> RecognitionUpdated
```

What is different from `SetAllResult`?

Reply:
Changed RecognitionUpdated to RecognitionResultUpdated.

Reply:
**RecognitionResult** - Receives command result registered by VC manager.
**SetAllResult** - Receives command result registered by VC Client.

VC manager himself can register VC command and receive matching result.
Usually, the system command is registered in the VC manager.
(Turn off, adjust the brightness, Turn up the volume)
So RecognitionUpdata exists.